client/src/components/layout/DogShowPage.js

```diff
@@ -1,10 +1,19 @@
 import React, { useState, useEffect } from "react";
 import Link from "react-dom";
+import ReviewTile from "./ReviewTile";
+import Fetcher from "./services/Fetcher.js";
```

Comment:
It looks like we're starting to see a mix of using this service, and writing out the fetch requests. As per Nick's previous comment, if you keep this `Fetcher` service it should be used for all fetch requests

Reply:
Where this is the only instance of a regular fetch request that I can see in this in this PR, could we make the refactor now so it's not lost and forgotten later?

Reply:
Refactor to use in the get on dogshowpage